File: FrontEnd/ideaevaluationfe/src/app/models/idea.ts
export class Idea{
    ideaId:number;
    ideaName:string;
    ideaDescription:string;
    evaluatedCount:number;

    constructor(ideaId: number, ideaName: string, description: string, count: number){
        this.ideaId = ideaId;
        this.ideaName = ideaName;
        this.ideaDescription = description;
        this.evaluatedCount = count;
    }
}

File: IdeaEvaluation/IdeaEvaluation/Models/Idea.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace IdeaEvaluation.Models
{
    public class Idea
    {
        public int IdeaId { get; set; }
        public string IdeaName { get; set; }
        public string IdeaDescription { get; set; }
        public int EvaluatedCount { get; set; }

    }
}